internal/server/v1/page/golden/place_page/county.Overview.json

```diff
@@ -22471,12 +22471,12 @@
       }
     },
     "geoId/06": {
-      "date": "2021",
-      "value": 39237836,
+      "date": "2022",
+      "value": 39029300,
       "metadata": {
-        "importName": "USCensusPEP_Annual_Population",
-        "provenanceUrl": "https://www2.census.gov/programs-surveys/popest/tables",
-        "measurementMethod": "CensusPEPSurvey",
+        "importName": "OECDRegionalDemography",
+        "provenanceUrl": "https://stats.oecd.org/Index.aspx?DataSetCode=REGION_DEMOGR#",
+        "measurementMethod": "OECDRegionalStatistics",
         "observationPeriod": "P1Y"
       }
```

Comment:
We might reconsider this... whether we return OECDRegionalDemography population data for US country and states.   Might also have search implications.

Reply:
Or maybe we need to update USCensusPEP?

Reply:
It is already at the top - https://github.com/datacommonsorg/mixer/blob/d7022933628a6df248842981d3cc75cbad697b8e/internal/server/ranking/ranking.go#L45-L48.  Any ideas why we OECD shows up for US places?

Reply:
When choosing the population series in place page, the facet with latest data is used indeed: https://github.com/datacommonsorg/mixer/blob/d7022933628a6df248842981d3cc75cbad697b8e/internal/server/v1/page/place.go#L331
Shall we not doing this?

Reply:
Sent a PR to test this out: https://github.com/datacommonsorg/mixer/pull/1275/files

Reply:
The PR is merged. @chejennifer can you sync and refresh the golden files?

Reply:
updated, ptal